src/main/java/com/stripe/net/StripeResponseStream.java

```diff
@@ -0,0 +1,30 @@
+package com.stripe.net;
+
+import com.stripe.util.StreamUtils;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class StripeResponseStream extends AbstractStripeResponse<InputStream> {
```

Comment:
Nit: StripeResponseStream -> StripeStreamResponse for consistency with other libraries?